velox/exec/fuzzer/ToSQLUtil.cpp

```diff
@@ -62,12 +105,97 @@ void toCallInputsSql(
 
 std::string toCallSql(const core::CallTypedExprPtr& call) {
   std::stringstream sql;
-  sql << call->name() << "(";
-  toCallInputsSql(call->inputs(), sql);
+  // Some functions require special SQL syntax, so handle them first.
+  if (call->name() == "in") {
+    VELOX_CHECK_EQ(call->inputs().size(), 2);
+    toCallInputsSql({call->inputs()[0]}, sql);
+    sql << " in (";
+    toCallInputsSql({call->inputs()[1]}, sql);
+    sql << ")";
+  } else if (call->name() == "like") {
+    toCallInputsSql({call->inputs()[0]}, sql);
+    sql << " like ";
+    toCallInputsSql({call->inputs()[1]}, sql);
+    if (call->inputs().size() == 3) {
+      sql << " escape ";
+      toCallInputsSql({call->inputs()[2]}, sql);
+    }
+  } else if (call->name() == "or" || call->name() == "and") {
+    sql << "(";
+    const auto& inputs = call->inputs();
+    for (auto i = 0; i < inputs.size(); ++i) {
+      if (i > 0) {
+        sql << fmt::format(" {} ", call->name());
+      }
+      toCallInputsSql({inputs[i]}, sql);
+    }
+    sql << ")";
+  } else if (call->name() == "array_constructor") {
+    sql << "ARRAY[";
+    toCallInputsSql(call->inputs(), sql);
+    sql << "]";
+  } else {
+    // Regular function call syntax.
+    sql << call->name() << "(";
+    toCallInputsSql(call->inputs(), sql);
+    sql << ")";
+  }
+  return sql.str();
+}
+
+std::string toCastSql(const core::CastTypedExprPtr& cast) {
+  std::stringstream sql;
+  if (cast->nullOnFailure()) {
+    sql << "try_cast(";
+  } else {
+    sql << "cast(";
+  }
+  toCallInputsSql(cast->inputs(), sql);
+  sql << " as " << toTypeSql(cast->type());
+  sql << ")";
+  return sql.str();
+}
+
+std::string toConcatSql(const core::ConcatTypedExprPtr& concat) {
+  std::stringstream sql;
```

Comment:
The logic are the same, but toConcatSql() cannot be merged into toCallSql() because ITypedExpr and ConcatTypedExpr don't have the name() method used in toCallSql(). So I think it's better to have them separated. Let me know if you have other suggestions.